stories/InstantSearch.stories.js

```diff
@@ -41,7 +41,7 @@ stories
           Root: 'div',
           props: {
             style: {
-              background: 'linear-gradient(80deg, #00D8FF, #00A7FF)',
+              padding: '30px',
```

Comment:
what about: `border: 1em solid red`? that will still be very easy to see, but will also show up in Argos

Reply:
Great idea!